frontend/src/views/groupmember/AddGroupMember.vue

```diff
@@ -283,17 +275,10 @@ export default {
       },
       homeAddress: {
         addressLine1: { required },
-        addressLine2: {},
-        addressLine3: {},
-        addressLine4: {},
-        postalCode: { required },
-      },
-      mailingAddress: {
-        addressLine1: {},
-        addressLine2: {},
-        addressLine3: {},
-        addressLine4: {},
-        postalCode: {},
```

Comment:
As discussed we should leave this in since they will be required for length/invalid character validation.